src/Grids/Grids.jl

```diff
@@ -92,5 +106,6 @@ include("grid_utils.jl")
 include("input_validation.jl")
 include("regular_cartesian_grid.jl")
 include("vertically_stretched_cartesian_grid.jl")
```

Comment:
This name will be changed too I suppose

Reply:
I think on #1377 we're discussing replacing "cartesian" with "rectilinear"